packages/components/src/Clickable/Clickable.tsx

```diff
@@ -30,17 +30,19 @@ export type ClickableProps<IComponent extends React.ElementType> = {
  *
  * See the Button stories for usage examples.
  */
-function Clickable<IComponent extends React.ElementType>({
-  as,
-  children,
-  color,
-  size,
-  state,
-  variant,
-  className,
-  ...rest
-}: ClickableProps<IComponent>) {
-  const Component = as;
+const Clickable = React.forwardRef(function <
+  IComponent extends React.ElementType,
+>(props: ClickableProps<IComponent>, ref: React.ForwardedRef<HTMLElement>) {
```

Comment:
Odd formatting by prettier (which comes with the territory of using it sometimes).

Chose to use a regular function instead of arrow here so that prettier would keep the indentation of the function body as-is, so I don't have to touch every line of this component.